infra-as-code/bicep/modules/hubNetworking/parameters/hubNetworking.parameters.all.json

```diff
@@ -93,11 +96,14 @@
         "privatelink.cassandra.cosmos.azure.com",
         "privatelink.gremlin.cosmos.azure.com",
         "privatelink.table.cosmos.azure.com",
+        "privatelink.eastus.batch.azure.com",
         "privatelink.postgres.database.azure.com",
         "privatelink.mysql.database.azure.com",
         "privatelink.mariadb.database.azure.com",
         "privatelink.vaultcore.azure.net",
         "privatelink.managedhsm.azure.net",
+        "privatelink.eastus.azmk8s.io",
+        "privatelink.eastus.backup.windowsazure.com",
```

Comment:
Why `eastus`? Do we need to add a comment or something saying this in this file incase they dont see the readme?

Reply:
This is due to copy/paste + find/replace. The bicep template contains these zones as the default value so I was bringing parameter file up to parity. However, the bicep template uses string interpolation to account for the region.

To be safe, I think it might be best to remove region specific zones from the parameters file until we can discuss a better way of handling.